config/Dockerfiles/JenkinsfileStageMerge

```diff
@@ -16,6 +16,33 @@ library identifier: "ci-pipeline@master",
                               remote: "https://github.com/CentOS-Paas-SIG/ci-pipeline"
 ])
 
+properties([
+  buildDiscarder(logRotator(artifactNumToKeepStr: '20', numToKeepStr: '20')),
+  [$class: 'GithubProjectProperty', displayName: '', projectUrlStr: 'https://github.com/CentOS-PaaS-SIG/linchpin/'],
+  [$class: 'org.jenkinsci.plugins.workflow.job.properties.PipelineTriggersJobProperty', triggers:[
+    [
+      $class: 'org.jenkinsci.plugins.ghprb.GhprbTrigger',
+      orgslist: 'CentOS-PaaS-SIG',
+      cron: 'H/5 * * * *',
```

Comment:
Why are you also using CRON?

Reply:
Shouldn't you only need the keyword only?

Reply:
ok

Reply:
What do you mean by "only need the keyword only?"

Reply:
like [test]  so you are not polling but now your explanation on minishift makes sense